model/version.go

```diff
@@ -107,8 +107,7 @@ func (v *Version) AddSatisfiedTrigger(definitionID string) error {
 // GetTimeSpent returns the total time_taken and makespan of a version for
 // each task that has finished running
 func (v *Version) GetTimeSpent() (time.Duration, time.Duration, error) {
-	// Find excludes display tasks from consideration
-	tasks, err := task.Find(task.ByVersion(v.Id).WithFields(task.TimeTakenKey, task.StartTimeKey, task.FinishTimeKey))
+	tasks, err := task.FindAllNewOrOld(task.ByVersion(v.Id).WithFields(task.TimeTakenKey, task.StartTimeKey, task.FinishTimeKey, task.DisplayOnlyKey), 0)
```

Comment:
I think it makes more sense to sum the durations for the latest execution of the task, rather than the first. My thinking is that because the latest is what we show by default, it's more what one would think when thinking about the makespan. Also if these tasks were restarted due to system failing or something, the latest is more likely to be the one that actually ran to completion

Reply:
I hear what you're saying with regard to time taken. 

On the other hand, it seems confusing for makespan and time taken to not refer to the same executions. An example of a confusing situation is if the makespan ends up longer than time taken because the first execution of a task was really long.